### PR TITLE
[nri-prometheus] adias/add worker threads config

### PR DESCRIPTION
**Pull request by ardias**
#### Is this a new chart
No

#### What this PR does / why we need it:
adds missing config options

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Review by davidbrota — APPROVED** [review]

File: charts/nri-prometheus/values.yaml [hunk 1]
```diff
@@ -68,6 +68,14 @@ config:
   # scrape_enabled_label: "prometheus.io/scrape"
   # Wether k8s nodes needs to be labelled to be scraped or not. Defaults to false.
   require_scrape_enabled_label_for_nodes: true
+  # Wether the integraton sends metrics directly to New Relic (true) or to the Infrastructure agent (false)
+  # standalone: false
+  # List of emitters to use. Valid values include: 'stdout', 'telemetry',' infra-sdk'
+  # Each emitter outputs data in a different format. Use 'telemetry' if using 'standalone = true', 'infra-sdk' if 'standalone = false'. 'stdout' is for debug purposes
+  # emmiters:
+  #   - "infra-sdk"
```
Comment (gsanchezgavier) [inline charts/nri-prometheus/values.yaml:76]: is it not pomi running standalone with the telemetry sdk always in k8s?

Comment (gsanchezgavier) [reply charts/nri-prometheus/values.yaml:76]: if so i would remove this comments

**Review by gsanchezgavier — APPROVED** [review]
looks good just small question

**Review by paologallinaharbur — APPROVED** [review]

File: charts/nri-prometheus/values.yaml (hunk 1 above)
Comment (ardias) [reply charts/nri-prometheus/values.yaml:76]: I'm not sure. Did you change that to set standalone automatically?

Comment (ardias) [reply charts/nri-prometheus/values.yaml]: ok, removed the emitters config. The code sets them based on standalone

**Pull request merged**